com.bardsoftware.constable/client/src/main/kotlin/com/bardsoftware/constable/client/ConstableClient.kt

```diff
@@ -0,0 +1,24 @@
+package com.bardsoftware.constable.client
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import java.sql.SQLException
+import java.util.logging.Level
+import java.util.logging.Logger
+
+private val logger = Logger.getLogger(H2Manager::class.java.name)
+
+class ConstableClient : CliktCommand() {
+    private val path by argument(help = "Path to database file")
```

Comment:
Will `file()`[https://ajalt.github.io/clikt/api/clikt/com.github.ajalt.clikt.parameters.types/file/] method work for this (to make sure it is really a file) ?

Reply:
Also, this database file seems to be unused. I mean, you can run `SELECT 2+2` query even when there is no file at all. I understand it is just the first baby step, but may be pass the command in another command line argument, execute it and save the file when command is comleted? So that we knew that our file is actually used and keeps the database state.

Reply:
Actually, path is path + database name as it is in URL: `jdbc:h2:[<path>]<databaseName>`, it could be splitted into 2 arguments.

Reply:
That's what shall be actually printed in the help string. Because now the help string says "path to the database file" which unambiguously means it is just a file.